s3compatapi/src/test/java/com/snowflake/s3compatapitestsuite/compatapi/S3CompatApiTest.java

```diff
@@ -295,7 +295,7 @@ void listNextBatchOfVersions() throws IOException {
         updatePrefixForTestCase(TestUtils.OPERATIONS.LIST_VERSIONS);
         // upload same file multiple times, to generate different versions
         File file = new File(EnvConstants.LOCAL_FILE_PATH_2);
-        String filePath = prefix + "/" + EnvConstants.LOCAL_FILE_PATH_2;
+        String filePath = prefix + "/foo%_foo%/" + EnvConstants.LOCAL_FILE_PATH_2;
```

Comment:
Is this intentional? If so, can you add a comment on what this string is used for?

Reply:
Yes. This is intentional. The `%` is used to trigger url encoding. 
When we specify url encoding for the request, pureStorage and s3 behaves the same if there is no special character, while Dell behaves slightly differently, it would encode `/`.
For String `/foo_foo/`, s3 and pureStorage encode it as `/foo_foo/`, Dell decodes it is as `%2Ffoo_foo%2F`
For String `/foo%_foo%/`, the 3 storage providers encode the `%`  as `%25`.
So I add `%` to trigger encoding from the request response.
I have updated the comment :)

Reply:
Got it, thanks.